server/http.go

```diff
@@ -534,6 +548,20 @@ func (p *Plugin) handleStartMeeting(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	restrict, statusCode, err := p.checkChannelPreference(req.ChannelID)
+	if err != nil {
+		p.API.LogError("Unable to check channel preference", "ChannelID", req.ChannelID, "Error", err.Error())
+		http.Error(w, err.Error(), statusCode)
+		return
+	}
+
+	if restrict {
+		if _, err = w.Write([]byte(`{"error": "Creating zoom meeting is disabled for this channel."}`)); err != nil {
```

Comment:
I know this manual JSON string pattern is already made throughout the project, but what do you think about this instead?

```go
type ErrorResponse struct {
    Error string `json:"error"`
}
```

```suggestion
		if err = json.NewEncoder(w).Encode(ErrorResponse{"Creating zoom meeting is disabled for this channel."}); err != nil {
```